test/test_yamllint_checks.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2024 Open Source Robotics Foundation, Inc.
+# Licensed under the Apache License, Version 2.0
+
+from pathlib import Path
+from typing import Iterable
+
+from git import Repo
+import pytest
+from rosdistro_reviewer.element_analyzer.yamllint import YamllintAnalyzer
+from rosdistro_reviewer.review import Recommendation
+
+# The control prefix intentionally contains a violation
+# to verify that we only surface problems related to new lines
+CONTROL_PREFIX = 'alpha: \n  bravo: charlie\n'
+CONTROL_SUFFIX = 'yankee:\n  - zulu\n'
+
+
+@pytest.fixture
+def empty_repo(tmp_path) -> Iterable[Repo]:
```

Comment:
Future reference maybe these fixtures could be deduplicated.

Reply:
Done in a26215f7c0d8